#10025 Removed global nested disabled styling from FileUpload

jepsar opened the pull request:
Fixes #10025

jepsar committed:
#10025 Removed global nested disabled styling

tandraschko commented on primefaces/src/main/resources/META-INF/resources/primefaces/fileupload/fileupload.css:
```diff
@@ -94,11 +94,6 @@
-    filter: alpha(opacity=100);
-}
-
 .ui-state-disabled input {
```
thats also global, should be removed too

jepsar committed:
Removed disabled input
Covered by .ui-fileupload-buttonbar .ui-fileupload-choose.ui-state-disabled input
